src/JSON For mIRC.mrc

```diff
@@ -456,7 +456,7 @@ alias JSONHttpFetch {
   }
 
   ;; Validate specified bvar when applicatable
-  elseif (b isincs %Switches) && (&* !iswm $2 || $0 > 2) {
+  elseif (b isincs %Switches) && (&* !iswm $2) || ($0 > 2) {
```

Comment:
This is incorrect; the correct version would be, the following but I opted not to have the duplicate `()`:
```
elseif (b isincs %Switches) && ((&* !iswm $2) || ($0 > 2)) {
```